source/resources/js/app/models/group.js

```diff
@@ -13,7 +13,18 @@ export default Backbone.Model.extend({
             });
         }
     },
+
     defaults: {
         'title': 'Untitled'
+    },
+
+    isTrash: function() {
+        let attr = this.get("attributes");
+
+        if (attr && "bc_group_role" in attr && attr.bc_group_role === "trash") {
```

Comment:
I understand this is not pretty, but don't know what would be test best way to handle it. Any suggestions?